spec/user_interface_spec.rb

```diff
@@ -290,6 +291,192 @@
     end
   end
 
+  describe '#choose_contact' do
+    let(:input) { StringIO.new("0\n") }
+
+    it 'prints all the contacts with an index' do
+      ui = described_class.new(input, output, validator)
+
+      contact_a = {
+        name: 'Matt Damon'
+      }
+      contact_b = {
+        name: 'John Doe'
+      }
```

Comment:
Up to you, but you can put these in the same line:
```suggestion
      contact_a = { name: 'Matt Damon' }
      contact_b = { name: 'John Doe' }
```